### PR TITLE
ibus-bamboo: add at 0.8.4

### PR DESCRIPTION
**Pull request by minhhut**
**Summary**
- add ibus-bamboo at v0.8.4 (https://github.com/BambooEngine/ibus-bamboo/releases/tag/v0.8.4-RC1)
- known issue with Wayland native (see BambooEngine/ibus-bamboo#44)

**Test Plan**

- Install ibus and run "IBus Preferences"
- Add Vietnamese - Bamboo from Input Method tab
- Add "Keyboard Layouts" plugin/widge to see the input switch icon
- Play around with language switch and many options from IBus Bamboo popup menu
- Focus on text input from browser, terminal or other app
- Type "xin chào các bạn"

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/i/ibus-bamboo/MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This file is used to indicate primary maintainership for this package. A package may list more than one maintainer to avoid bus factor issues. People on this list may be considered “subject-matter experts”. Please note that Solus Staff may need to perform necessary rebuilds, upgrades, or security fixes as part of the normal maintenance of the Solus package repository. If you believe this package requires an update, follow documentation from https://help.getsol.us/docs/packaging/procedures/request-a-package-update. In the event that this package becomes insufficiently maintained, the Solus Staff reserves the right to request a new maintainer, or deprecate and remove this package from the repository entirely.
+
+- Name Surname
```
Comment (malfisya) [inline packages/i/ibus-bamboo/MAINTAINERS.md:3]: This should be replaced by your (actual) name: Minh Nguyen

File: packages/i/ibus-bamboo/package.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+license    : GPL-3.0-or-later
+component  : desktop.core
+summary    : Vietnamese Input Method Engine for IBus using Bamboo Engine
+description: |
+    Vietnamese Input Method Engine for IBus using Bamboo Engine
+builddeps  :
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(ibus-1.0)    
+    - golang
+    - libx11-devel
```
Comment (EbonJaeger) [inline packages/i/ibus-bamboo/package.yml:16]: Please replace with `pkgconfig(x11)`

File: packages/i/ibus-bamboo/package.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    - libxtst-devel
```
Comment (EbonJaeger) [inline packages/i/ibus-bamboo/package.yml:17]: Please replace with `pkgconfig(xtst)`

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
We prefer to use pkgconfig names for dependencies whenever possible. :)

Also, could you please squash your commits so that there is only one commit for the package? Thanks!

**Comment by minhhut** [comment]
Updated as review's comments. Thanks for the hints!

**Review by EbonJaeger — APPROVED** [review]
LGTM! Thanks for the contribution! :)

**Pull request merged**